sign_verify_file_ssh.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+
```

Comment:
Would be nice if the script has some standard header with

- short note about purpose,
- author(s)
- license